lib/include/public/Version.hpp

```diff
@@ -18,7 +18,7 @@ namespace MAT_NS_BEGIN {
 uint64_t const Version =
     ((uint64_t)3 << 48) |
     ((uint64_t)7 << 32) |
-    ((uint64_t)46 << 16) |
+    ((uint64_t)62 << 16) |
```

Comment:
nit - do you plan to do new release with this version, or else you want me to do a release?

Reply:
Yes, I can do a release.